tvb_epilepsy/base/analyzers_factory.py

```diff
@@ -0,0 +1,188 @@
+import numpy as np
+from scipy.signal import butter, lfilter, welch, periodogram
+
+# x is assumed to be data (real numbers) arranged along the first dimension of an ndarray
+# this factory makes use of the numpy array properties
+
+
+# Pointwise analyzers:
+
+
+def center(x, cntr=0.0):
+    return (x.T-np.array(cntr)).T
+
```

Comment:
This modules is improperly named "_factory".
I see no Factory pattern implemented here, but rather gathering of many helper functions.